src/api/api.cpp

```diff
@@ -349,7 +349,7 @@ rgl_graph_get_result_data(rgl_node_t node, rgl_field_t field, void* data)
 		}
```

Comment:
Since FormatPointsNode is `IPointsNode` if statement above is not needed. Could be simplified to:
```suggestion
auto pointCloudNode = Node::validatePtr<IPointsNode>(node);
VArray::ConstPtr output = pointCloudNode->getFieldData(field, nullptr);
		}
```

Reply:
Good point. The suggested change seems wrong but I applied it manually.